dialogTemplate/de.json

```diff
@@ -179,30 +149,20 @@
 				{
 					"value": "stop",
 					"synonyms": [
-						"cancel",
-						"stopp",
-						"abbrechen",
-						"brech",
-						"breche",
-						"stoppe",
-						"halte",
-						"aus"
+						"stornieren"
```

Comment:
different cases missing like "storniere", more words exist for german "stoppe", "breche ab" etc.